packages/feedback/.eslintignore

```diff
@@ -0,0 +1,6 @@
+node_modules/
+build/
+demo/build/
+# TODO: Check if we can re-introduce linting in demo
+demo
+metrics
```

Comment:
```suggestion
```

we actually don't need any of these (and should also remove them from the replay package 😅 )